mlflow/sagemaker/__init__.py

```diff
@@ -1237,6 +1237,16 @@ def _get_deployment_config(flavor_name):
         DEPLOYMENT_CONFIG_KEY_FLAVOR_NAME: flavor_name,
         SERVING_ENVIRONMENT: SAGEMAKER_SERVING_ENVIRONMENT,
     }
+
```

Comment:
@harupy and @WeichenXu123 - Should I proceed with the proxy conditional checks and then only update the `deployment config`, or should I default all proxies to 'empty strings' if proxies are not available? I'm asking because I'd like to change a couple of unit test cases based on your input. Thanks.

Reply:
I prefer to your current approach: if no proxy related setting, it do not add related item to `deployment_config`